packages/framework/src/Resources/config/services.yaml

```diff
@@ -777,6 +777,10 @@ services:
             - '%env(MAILER_DSN)%'
             - '%env(bool:MAILER_FORCE_WHITELIST)%'
 
+    Shopsys\FrameworkBundle\Model\Mail\MailTemplateSender\MailTemplateSenderFacade:
+        arguments:
+            $mailTemplateSenders: !tagged shopsys.mail_template_sender
```

Comment:
we should use `!tagged_iterator` for future compatibility
https://symfony.com/blog/new-in-symfony-7-2-deprecations#deprecated-the-tagged-tag

Reply:
nice :ok_hand: 7baa7349eb